src/js/Node.js

```diff
@@ -1908,6 +1910,33 @@ Node.prototype._getDomField = function(silent) {
   }
 };
 
+/**
+ * Update the value of the schema default element in the DOM.
+ * @private
+ * @returns {undefined}
+ */
+Node.prototype._updateDomDefault = function () {
+  // Short-circuit if schema is missing, has no default, or if Node has children
+  if (!this.schema || !this.schema.default || this._hasChilds()) {
+    return;
+  }
+
+  if (this.value === this.schema.default) {
```

Comment:
Since you are setting all these things it would be nice if they are all ordered the same. Instead of title being set below text-context in the first if and at the top of the second if...